.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Similar to a comment made on a different PR by @yuzilyu , `.vscode/settings.json` should be included in our `.gitignore`. Recent changes to `main` should have this, so we should sync this branch before merging